# config/pcl_map_voxel_grid_downsample.param.yaml
---

/**:
  ros__parameters:
    is_approximate: false
    publisher:
      qos:
        history_depth: 3
        durability: transient_local
    subscription:
      qos:
        history_depth: 3
        durability: transient_local
    config:
      capacity: 10000000
      min_point:
        x: -1300000.0
        y: -1300000.0
        z: -3.0
      max_point:
        x: 13000000.0
        y: 13000000.0
        z: 3.0
      voxel_size:
        x: 0.2
        y: 0.2
        z: 0.2
